src/Mvc/Mvc.Core/src/Infrastructure/ObjectResultExecutor.cs

```diff
@@ -105,7 +105,7 @@ public virtual Task ExecuteAsync(ActionContext context, ObjectResult result)
             if (selectedFormatter == null)
             {
                 // No formatter supports this.
-                Logger.NoFormatter(formatterContext);
```

Comment:
So one thing to mention about this is that the list logged here won't include anything from the accept header. The logic to iterate the accept header (or not in the case of `result.ContentType`) lives inside the formatter selector. What's the logging like when you have an accept header for `application/xml` and only have the JSON formatter registered?

Reply:
> So one thing to mention about this is that the list logged here won't include anything from the accept header.

That's right, but there are other (debug-level) log statements that include the accept header:

> Attempting to select an output formatter based on Accept header '{AcceptHeader}'.
> Attempting to select an output formatter without using a content type as no explicit content types were specified for the response.

or

> Attempting to select an output formatter based on Accept header '{AcceptHeader}' and explicitly specified content types '{ExplicitContentTypes}'. The content types in the accept header must be a subset of the explicitly set content types.
> Attempting to select the first output formatter in the output formatters list which supports a content type from the explicitly specified content types '{ExplicitContentTypes}'.

depending on whether `result.ContentTypes` is set.

@rynowak Do you think Accept-header values should be included in this (`NoFormatter`) statement as well?

Reply:
This seems ok then. It's an improvment, if we need to improve it more in the future we can.